packit_service/worker/handler.py

```diff
@@ -116,9 +111,9 @@ def report(
         self,
         state: str,
         description: str,
+        check_names: Union[str, list, None],
```

Comment:
Maybe more descriptive:

```suggestion
        check_names: Opotional[Union[str, list]],
```